realm/realm-annotations-processor/build.gradle

```diff
@@ -0,0 +1,56 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+apply plugin: 'com.github.johnrengelman.shadow'
+
+sourceCompatibility = '1.6'
+targetCompatibility = '1.6'
+
+dependencies {
+    compile group:'com.squareup', name:'javawriter', version:'2.5.0'
+    compile project(':realm-annotations')
+    testCompile files('../realm-library/build/intermediates/bundles/release/classes.jar') // Java projects cannot depend on AAR files
+    testCompile files("${System.properties['java.home']}/../lib/tools.jar") // This is needed otherwise compile-testing won't be able to find it
+    testCompile group:'junit', name:'junit', version:'4.11'
+    testCompile group:'com.google.testing.compile', name:'compile-testing', version:'0.6'
+    testCompile files(file("${System.env.ANDROID_HOME}/platforms/android-21/android.jar"))
```

Comment:
I tried and the combination of Gradle + SdkManager plugin explodes on me so... no :(